selfdrive/controls/radard.py

```diff
@@ -218,26 +229,32 @@ def update(self, sm: messaging.SubMaster, rr: Optional[car.RadarData]):
     if sm.updated['modelV2']:
       self.ready = True
 
-    ar_pts = {}
-    for pt in radar_points:
-      ar_pts[pt.trackId] = [pt.dRel, pt.yRel, pt.vRel, pt.measured]
+    if not no_radar:
```

Comment:
not not always makes my had spin. Why not just give rr a proper name and do "If RadarSomething is not None:"